code/game/objects/objs.dm

```diff
@@ -307,12 +307,12 @@ a {
 		return TRUE
 	return FALSE
 
-/obj/water_act(volume, temperature, source, method = REAGENT_TOUCH)
+/obj/water_act(volume = 0, temperature = 0, source, method = REAGENT_TOUCH)
 	. = ..()
 	extinguish()
 	acid_level = 0
 
-/obj/singularity_pull(S, current_size)
+/obj/singularity_pull(S, current_size = 0)
```

Comment:
The default here does not make much sense. As the singulo should always pass it's size